<launch>
	<arg
		name="t_device_name"
		default="tecan_xcalibur"/>
	<arg
		name="t_connection_mode"
		default="serial"/>
	<arg
		name="t_address"
		default="None"/>
	<arg
		name="t_port"
		default="/dev/ttyUSB0"/>
	<arg
		name="t_switch_address"
		default="0"/>
	<arg
		name="t_syringe_size"
		default="1.0"/>
	<arg
		name="t_simulation"
		default="False"/>
	<node
		name="tecan_xcalibur"
		pkg="roslabware_drivers"
		type="tecan_xcalibur"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)
			-w $(arg switch_address)
			-z $(arg syringe_size)"
		output="screen"/>
</launch>
